.github/workflows/security-scan.yml

```
@@ -0,0 +1,218 @@
+      alerts_low: ${{ steps.report.outputs.alerts_low }}
+      alerts_medium: ${{ steps.report.outputs.alerts_medium }}
+      alerts_high: ${{ steps.report.outputs.alerts_high }}
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Setup dependencies
+      run: |
+        npm ci
+
+    - name: Run static security scan
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2261:error:1:23: Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead

(shellcheck)